app/src/main/java/de/danoeh/antennapod/spa/SPAUtil.java

```diff
@@ -35,7 +35,7 @@ private SPAUtil() {
      * sent before.
      */
     public static synchronized boolean sendSPAppsQueryFeedsIntent(@Nullable Context context) {
-        if (context == null) throw new IllegalArgumentException("context = null");
+        assert context != null : "context = null";
```

Comment:
Doesn't this change behavior? Are these assertions checked during runtime?

Reply:
Good catch @mfietz - this change was local when I was testing some other cleanup and should have been reverted. I've added a commit to take care of this.